### PR TITLE
[Spring security 6 migration] Add recipe to remove unnecessary `use-authorization-manager` for message security in Spring security 6

### PR DESCRIPTION
**Pull request by kunli2**
This recipe is part of [Spring Security 6.0 migration ](https://github.com/openrewrite/rewrite-spring/issues/256) , to remove unnecessary `use-authorization-manager` for message security in Spring security 6. 
reference: [Use AuthorizationManager for Message Security](https://docs.spring.io/spring-security/reference/6.0.0/migration/servlet/authorization.html#_use_authorizationmanager_for_message_security)

before
```xml
<websocket-message-broker use-authorization-manager="true"/>
```
after
```xml
<websocket-message-broker/>
```

### CODE REVIEW

File: src/testWithSpringSecurity_5_8/java/org/openrewrite/spring/security5/RemoveUseAuthorizationManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(Environment.builder()
+          .scanRuntimeClasspath()
+          .build()
+          .activateRecipes("org.openrewrite.java.spring.security6.RemoveUseAuthorizationManager"));
+    }
+
+    @Test
+    void removeUseAuthorizationManagerAttribute() {
```
Comment (sambsnyd) [inline src/testWithSpringSecurity_5_8/java/org/openrewrite/spring/security5/RemoveUseAuthorizationManagerTest.java:36]: Is this test necessary? Seems like it mostly tests `ChangeTagAttribute`, which already has its own tests.

File: src/main/resources/META-INF/rewrite/spring-security-60.yml [hunk 2]
```diff
@@ -33,3 +33,21 @@ recipeList:
   - org.openrewrite.java.spring.security6.UseSha256InRememberMe
   - org.openrewrite.java.spring.security6.PropagateAuthenticationServiceExceptions
   - org.openrewrite.java.spring.security6.RequireExplicitSavingOfSecurityContextRepository
+---
+########################################################################################################################
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.security6.RemoveUseAuthorizationManager
```
Comment (sambsnyd) [inline src/main/resources/META-INF/rewrite/spring-security-60.yml:39]: Should this be included as part of  the larger spring-security migration?

File: src/testWithSpringSecurity_5_8/java/org/openrewrite/spring/security5/RemoveUseAuthorizationManagerTest.java (hunk 1 above)
Comment (kunli2) [reply src/testWithSpringSecurity_5_8/java/org/openrewrite/spring/security5/RemoveUseAuthorizationManagerTest.java:36]: Yes, `ChangeTagAttribute` didn't support removing an attribute before and I just changed it to support it.
I think adding an example to demonstrate this specific use case is no harm.

File: src/main/resources/META-INF/rewrite/spring-security-60.yml (hunk 2 above)
Comment (kunli2) [reply src/main/resources/META-INF/rewrite/spring-security-60.yml:39]: Yes, going to add this to the spring-security migration recipe list.

Comment (kunli2) [reply src/main/resources/META-INF/rewrite/spring-security-60.yml:42]: added to the spring security 6 recipe list

**Pull request merged**